src/main/java/org/openmrs/addonindex/backend/Bintray.java

```diff
@@ -98,8 +131,21 @@ AddOnInfoAndVersions handlePackageJson(AddOnToIndex addOnToIndex, String package
 				}
 			}
 		}
+
+		String downloadCountUrl = downloadCountUrlFor(addOnToIndex);
+		JsonNode arr = restTemplateBuilder.build().getForObject(downloadCountUrl, JsonNode.class);
+		int totalDownloadCount = handleBintrayGeoStats(arr);
+		info.setDownloadCounts(totalDownloadCount);
 		return info;
 	}
+
+	private int handleBintrayGeoStats(JsonNode jsonNode) {
+		int totalDownloadCount = 0;
```

Comment:
I do see you've done this twice in two ways. I like the other way (using a POJO) because it's self-documenting, and it makes it clear what the response from that URL is going to be.